medusa/scene_exceptions.py

```diff
@@ -140,6 +140,12 @@ def get_scene_exceptions_by_name(show_name):
     # TODO: Rewrite to use exceptions_cache since there is no need to hit db.
     # TODO: Make the query more linient. For example. `Jojo's Bizarre Adventure Stardust Crusaders` will not match
     # while `Jojo's Bizarre Adventure - Stardust Crusaders` is available.
+    if show_name is None:
+        logger.debug(
+            'Scene exception lookup failed because no show name was provided'
+        )
```

Comment:
@medariox Maybe make this a one-liner?

Reply:
I'm okay with the change, but you'll need to push the change or someone else. Not at a PC until tomorrow.